kolibri/plugins/coach/assets/src/views/exams/create-exam-page/topic-row.vue

```diff
@@ -3,21 +3,20 @@
   <tr>
     <th class="core-table-checkbox-col">
       <k-checkbox
+        v-if="!channel"
         :label="$tr('selectTopic')"
         :showLabel="false"
         :checked="allExercisesWithinTopicSelected"
         :indeterminate="someExercisesWithinTopicSelected"
-        :disabled="!topicHasExercises"
         @change="changeSelection"
       />
     </th>
     <td class="core-table-main-col">
       <div class="topic-title">
         <content-icon :kind="topic" :class="{ disabled: !topicHasExercises }" />
-        <button v-if="topicHasExercises" class="title" @click="$emit('goToTopic', topicId)">
+        <button class="title" @click="$emit('goToTopic', topicId)">
```

Comment:
haven't tested yet .. but shouldn't the `v-if` remain here? I believe the decision was that that we want to completely hide items that don't have exercises

Reply:
oh nevermind - that's here https://github.com/learningequality/kolibri/pull/4173/files#diff-b4cc9678ab849b9bff180cbba2e98b45R91

Reply:
The hiding now happens at in `index.vue`, this `v-if` logic toggled the disabled state, so was no longer necessary if we are just completely hiding topics without exercises.